docs/ref/templates/builtins.txt

```diff
@@ -1463,7 +1463,11 @@ When used without a format string::
     {{ value|date }}
 
 ...the formatting string defined in the :setting:`DATE_FORMAT` setting will be
-used, without applying any localization.
+used.
+
+.. versionchanged:: 1.10
+
+    :setting:`USE_L10N` is honored even when no format string is given.
```

Comment:
I don't think it's clear from reading this note what behavior changed. Generally the new behavior would be described outside the versionchanged block and then this note might say, "In older versions, ..."